manifests/a/Audiveris/Audiveris/5.3.1/Audiveris.Audiveris.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Audiveris.Audiveris
+PackageVersion: 5.3.1
+PackageLocale: en-US
+Publisher: Audiveris

```

```suggestion
Publisher: Audiveris Team
```
This should match control panel